x-pack/plugins/security_solution/public/management/pages/trusted_apps/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const SEARCHABLE_FIELDS = [`name`, `description`, `entries.value`, `entries.entries.value`];
```

Comment:
can you make this readonly?  adding Type to it should be sufficient (`ReadOnly<string[]>`), but could also use `Object.freeze()`